### DISCUSSION TITLE
Some help installing and running the neolink in ubuntu Server

### DISCUSSION THREAD

**Discussion by nunokaneco** [thirtythreeforty/neolink#301] [Q&A]
Hello to all,

I just need to install and run the neolink in a Ubuntu Server machine:

I followed the instructions:

Ubuntu/Debian/Raspberry Pi OS example:
´´´
unzip release-arm64-buster.zip
sudo cp neolink /usr/local/bin/
sudo chmod +x /usr/local/bin/neolink
´´´
And I created the my_config.toml 

![image](https://user-images.githubusercontent.com/33290686/188847489-05c86c94-a3ff-4b64-a6fe-ab6ac493217a.png)

But when I try to run the command 
´´´
neolink rtsp --config config.toml 
´´´
I'm receiving the following error msg ...

![image](https://user-images.githubusercontent.com/33290686/188846288-f6e6de3f-1c31-4e45-a7f2-8cd785b81c09.png)

Can anyone please help me in this step ? Thank you for your Support and best regards

**Comment by QuantumEntangledAndy** [comment]
If you called it `my_config.toml` as you say then your command would be:

```bash
neolink rtsp --config my_config.toml
```

**Reply by nunokaneco** [reply]
Thank you for your fast reply,

I also try without success  
'''
neolink rtsp --config my_config.toml
'''
![image](https://user-images.githubusercontent.com/33290686/188849475-3943015f-1cc2-4d82-991a-5bd39ece0ee3.png)

But once I changed the name to the config.toml instead of the my_config.toml ....

**Reply by QuantumEntangledAndy** [reply]
Oh wait a second this is the loader. Ok you should not use `release-arm64-buster.zip` for an `x86_64` machine

Please download and install the x86_64 build  not the arm64 build. The arm build is for raspberrypis

**Reply by nunokaneco** [reply]
Ok I removed all the files, and I performed the download again now for the Ubuntu/Debian x86_64 

But I still facing this error:

![image](https://user-images.githubusercontent.com/33290686/188853068-652382cf-f8fa-4f0f-9961-1d41c02b56e5.png)

**Reply by QuantumEntangledAndy** [reply]
Please install the dependencies listed in the docs https://github.com/thirtythreeforty/neolink/blob/master/docs/unix_setup.md

```bash
sudo apt install \
  libgstrtspserver-1.0-0 \
  libgstreamer1.0-0 \
  libgstreamer-plugins-bad1.0-0 \
  gstreamer1.0-plugins-good \
  gstreamer1.0-plugins-bad
```

**Reply by nunokaneco** [reply]
Great thank you for your support now is working 

By the way can you please let me know the way to run the neolink as a service ?

**Reply by QuantumEntangledAndy** [reply]
Follow the documentation in the doc folder for unix_service
